content/docs/command-reference/remote/modify.md

```diff
@@ -298,6 +298,26 @@ For more information on configuring Azure Storage connection strings, visit
 
 <details>
 
+### Click for Dropbox
+
+Please see [Setup a Dropbox DVC Remote](/doc/user-guide/setup-dropbox-remote)
+for a full guide on using Dropbox as DVC remote storage.
+
+- `url` - remote location, in the `dropbox://<dvc-subdirectory>` format:
+
+  ```dvc
+  $ dvc remote modify myremote url dropbox://my-project
```

Comment:
qq - for multiple people to have access to this they need to have the same dir mounted under the same name. Let's say I share a dir in Dropbox, and someone already has `my-project` in their Dropbox how will it resolve the conflict?

Reply:
They would need to have it under the same name, but the path doesn't have to be as simple, nor one-level - one could define any nested subdirectory.